snakefmt/parser/syntax.py

```diff
@@ -125,31 +132,46 @@ def check_empty(self):
                 f"{self.line_nb}{self.keyword_name} has no keywords attached to it."
             )
 
+    @property
+    def effective_indent(self):
+        return max(0, self.cur_indent - self.target_indent)
+
     def get_next_queriable(self, snakefile):
         buffer = ""
-        newline = False
+        newline, used_name = False, True
         while True:
             token = next(snakefile)
             t_t = token.type
```

Comment:
prefer `token_type` to `t_t`

Reply:
i find it more readable this way

Reply:
tried changing it, and gets harder to forget about reading text around it

Reply:
There is no way `t_t` is more readable than `token_type`. In fact, we probably don't even need a variable as `token.type` is very descriptive already.  
I know it seems like I am being picky about inconsequential things but clean code is just as (if not more) important than code functionality. And this definitely isn't clean code.